jekyll/_cci2/writing-yaml.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: classic-docs
+title: "Writing YAML"
+short-title: "Writing YAML"
+description: "How to Write YAML on CircleCI"
+order: 20
+---
+
+*[Basics]({{ site.baseurl }}/2.0/basics/) > Writing YAML*
+
+This document gives an overview of YAML,
```

Comment:
gives...-> describes basic information about YAML and how to use anchors to prevent repeated YAML stanzas in your CircleCI configuration. (gives is too informal and should at least be 'gives you', but the goal of this article is to help people with long configuration files so saying that first is probably helpful.)

Reply:
👍 to the wording choice re: "gives".

I think I see the actual **goal** of the document differently: isn't this more about giving people 90% of the YAML formatting they need to write a `config.yml`? Only one section actually deals with configuration files being long, and that would be covered by the table of contents.

Something that still isn't clear to me is the difference between the **description** and **overview** of a document. Could you help me with some broad descriptions of both of those, and I'll get them added to the Wiki?